Formula/infer.rb

```diff
@@ -15,12 +15,21 @@ class Infer < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "camlp4" => :build
   depends_on "cmake" => :build
+  depends_on "gnu-sed" => :build
   depends_on :java => ["1.7+", :build]
   depends_on "libtool" => :build
   depends_on "ocaml" => :build
   depends_on "opam" => :build
+  depends_on "perl" => :build
```

Comment:
System perl won't work?

Reply:
Similar to `gnu-sed`, some of the ocaml libs use perl in their configure steps. I'll remove and see if still works.

Reply:
yes please. Thanks